merklize/merklize.go

```diff
@@ -1091,7 +1093,9 @@ func convertAnyToString(value any) (str string, err error) {
 	return str, nil
 }
 
-func convertStringToXSDValue(datatype string, value string) (resultValue interface{}, err error) {
+func convertStringToXSDValue(datatype string,
```

Comment:
Hard to read.
How about:
```
func convertStringToXSDValue(datatype, value string)...
```
or
```
func convertStringToXSDValue(
  datatype string
  value string
) (
result ...
)
```

Reply:
I think if a function has more than 3 arguments, that function is a bad function. And we should refactor them )